### PR TITLE
support tensorflow v2.x

### PR DESCRIPTION
**Pull request by acezen**

### CODE REVIEW

**Comment by acezen** [comment]
cc/ @sighingnow

File: examples/tf/bipartite_graphsage/train_unsupervised.py [hunk 1]
```diff
@@ -20,7 +20,12 @@
 
 import graphlearn as gl
 import numpy as np
-import tensorflow as tf
+itry:
```
Comment (sighingnow) [inline examples/tf/bipartite_graphsage/train_unsupervised.py:23]: `itry` -> `try`.

**Review by sighingnow — CHANGES_REQUESTED** [review]

**Comment by acezen** [comment]
cc/ @jackonan please take a review, thanks

**Review by baoleai — APPROVED** [review]

**Review by baoleai — APPROVED** [review]

**Comment by sighingnow** [comment]
Closed as merged into master.

**Pull request closed without merging**